python/tests/test_logistic_regression.py

```diff
@@ -463,13 +482,9 @@ def test_compat(
         )
 
         assert _LogisticRegression().getRegParam() == 0.0
-        if lr_types[0] is SparkLogisticRegression:
-            blor = _LogisticRegression(
-                regParam=0.1, fitIntercept=fit_intercept, standardization=False
-            )
-        else:
-            warnings.warn("spark rapids ml does not accept standardization")
-            blor = _LogisticRegression(regParam=0.1, fitIntercept=fit_intercept)
+        blor = _LogisticRegression(
```

Comment:
Should we now have a version of this test with standardization on?

Reply:
Yeah, revised to add standardization=True